### PR TITLE
Fixed the z-index of edit icon in link

### LINKED ISSUE #8507: [BUG] The edit-milestone-icon comes over the mobile menu.
### Description

The Z-index of the Edit-milestone-icon is greater than the mobile menu.

### Screenshots

image

### Additional information

I would like to work on this issue, could you assign it to me?

### PR DESCRIPTION
**Pull request by mishra-nikhil0110**
fixes #8507

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Screenshot (75)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8535"></a>

### CODE REVIEW

**Review by sital002 — COMMENTED** [review]
Can you provide the issue link so, we can know which issue are you solving.

**Comment by mishra-nikhil0110** [comment]
> Can you provide the issue link so, we can know which issue are you solving.

okk here is the link: 
https://github.com/EddieHubCommunity/LinkFree/issues/8507

**Comment by eddiejaoude** [comment]
> > Can you provide the issue link so, we can know which issue are you solving.
> 
> okk here is the link: #8507

I will edit the description to have this, to make it easier for everyone to find

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 

Changes in `Edit.js` look good, but you have blocked your PR because you included changes from another issues not assigned (as the maintainer already mentioned to you in the issue) to you plus breaks another page (see screenshot) - smaller focused pull requests are the best, please undo changes from `UserMilestone.js`

Screenshot 2023-08-07 at 07 54 26

File: components/user/UserMilestone.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ export default function UserMilestone({ milestone, isGoal, manage }) {
       : "text-primary-medium dark:text-primary-low-medium";
 
     return (
-      <div className="flex space-x-3 grow">
-        {milestone.icon && <DisplayIcon className="h-8 w-8 rounded-full" />}
+      <div className="flex space-x-2 grow">
+        {milestone.icon && <DisplayIcon className="h-8 w-8 ml-11"/>}
```
Comment (eddiejaoude) [inline components/user/UserMilestone.js:27]: These changes are not part of this pull request plus break another page where it is used, please remove these changes so we can merge your pull request

```suggestion
      <div className="flex space-x-3 grow">
        {milestone.icon && <DisplayIcon className="h-8 w-8 rounded-full" />}
```

Comment (mishra-nikhil0110) [reply components/user/UserMilestone.js:27]: okk. I'll discard changes suggested.👍

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**